### ISSUE TITLE
[feature request] multiselect for action

### ISSUE THREAD

**Issue by nr458h** [PeterCxy/Shelter#24]
In the setup process or sometimes later on it would be helpful to have the ability to apply an action to a multiselected number of apps.
For example multiselect some apps and than
- create a shortcut for them
- unfreeze and launch them
- freeze them
- untill issue with afwall isn't closed also clone to mainland would be helpful

**Comment by PeterCxy** [comment]
I'm not sure how some of the actions should be performed with multiple selections. For example, how should cloning multiple apps be implemented? There is no way to call PackageInstaller with multiple apps.

**Comment by nr458h** [comment]
True, but for that specific action multiselection is not that important to me (probably it is possible to use later root or adb for this action)

**Comment by chilliger** [comment]
I would like to have a mulitselect unfreeze or batch unfreeze short cut. That way it would be easier to update all apps through the an workprofile appstore like auroraStore.

**Comment by chilliger** [comment]
Solved the aurora problem. You have to use the root installion method and select the work profile user in the aurora intsallation settings. Or use the aurora service/magisk module installtion method.

@PeterCxy Is a shortcut for unfreezing all apps possible?

**Comment by dimqua** [comment]
> select the work profile user in the aurora intsallation settings

Please explain how to do this.

**Comment by chilliger** [comment]
![](https://i.imgur.com/zYBzLVQ.png)

**Comment by dimqua** [comment]
>Is a shortcut for unfreezing all apps possible?

for reference: https://git.angry.im/PeterCxy/Shelter/issues/16

**Comment by ioogithub** [comment]
My ROM is heavily bloated. I have diligently gone though and multi select all of the apps I wish to freeze. There are 155 apps on the list. Now when I click the three dots I can't freeze them? I have to select each one by one and freeze it? This would take me a week. Surely a batch operation can be made to freeze more than one app at a time??
